### PR TITLE
WIP: setup chromium headless for velum instead of phantomjs

### PR DESCRIPTION
**Pull request by MalloZup**
no worth for reviwiing atm 

#### Todos:

- [ ] enable on local-devenv mode too

### CODE REVIEW

File: .ci/chromium-setup.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#! /bin/bash
+
+set -e
+# this script install chromium headless for a debian system used in CI.
+snap install chromium
```
Comment (vitoravelino) [inline .ci/chromium-setup.sh]: Doesn't adding 

```
addons:
  chrome: stable
```

fix that?

Comment (MalloZup) [reply .ci/chromium-setup.sh]: @vitoravelino i'm trying to use `chromium` which is more updated atm.

Anywyas is kind wip, i need to find out the repository of latest chromium for ubuntu so it's compatible with the latest chrome-driver :racehorse:

Comment (MalloZup) [reply .ci/chromium-setup.sh]: thx @vitoravelino anyways for that suggestion

Comment (vitoravelino) [reply .ci/chromium-setup.sh]: No problem. :wink:

**Comment by MalloZup** [comment]
@ereslibre seems that Travis kill the job because to much time. Can you run this locally and post the output here if you have 3 min time? I think locally the job will not killed. Chrome and the driver are good setup but I cannot see where it is failing.

**Comment by vitoravelino** [comment]
Try to decrease the capybara default wait time. It's probably that.

**Comment by MalloZup** [comment]
Thx Vito for hint. 🎸

**Comment by bear454** [comment]
Maybe you can derive some value from https://github.com/openSUSE/osem/pull/2225/files .

**Comment by MalloZup** [comment]
This GitHub PR is unactive since more then 30 days. Is this GitHub PR still needed? Please close or update it accordingly.
         This reminder is autogenerated by https://github.com/MalloZup/blacktango

**Comment by thehejik** [comment]
@MalloZup I don't think we need it, please close

**Comment by MalloZup** [comment]
@thehejik  yes, i think we can close thank you for your help here tomas. 

thx all for feedback and help.

**Pull request closed without merging**